Zend/tests/partial_application/variation_variadics_007.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+Partial application variation extra through variadic
+--FILE--
+<?php
+function appliesCb($cb) { return runsCb($cb(1, ?, ?, ...)); } 
```

Comment:
The behavior here is inconsistent. This would throw an error about two many placeholders if you wrote `$cb(1, ?, ?)`. It doesn't make sense to me that `$cb(1, ?, ?, ...)` doesn't throw the same error.

Reply:
I simply misread this code yesterday ... fixed.